bigcommerce/resources/products.py

```diff
@@ -48,11 +48,17 @@ def rules(self, id=None):
         else:
             return ProductRules.all(self.id, connection=self._connection)
 
-    def skus(self, id=None):
+    def skus(self, id=None, **kwargs):
         if id:
-            return ProductSkus.get(self.id, id, connection=self._connection)
+            return ProductSkus.get(self.id, id, connection=self._connection, **kwargs)
         else:
-            return ProductSkus.all(self.id, connection=self._connection)
+            return ProductSkus.all(self.id, connection=self._connection, **kwargs)
+
+    def variants(self, id=None, **kwargs):
```

Comment:
I would prefer to have the V3 products-related endpoints in a completely separate file from the V2 ones - that namespacing will help a lot if we try to apply different logic to them (pagination comes to mind)